src/components/onboarding/PaymentForm.tsx

```diff
@@ -41,7 +50,11 @@ export const OnboardingPaymentForm: React.FC<OnboardingPaymentForm> = ({
     const { error: submitError } = await elements.submit();
```

Comment:
actually not new code but above this line where we do
```
if (!stripe || !elements) {
      return;
    }
```
    
this means if stripe or elements is not available we never re-set `setSaving` and the user will be stuck - we should move the `setSaving(true)` below the early return statement

Reply:
Great thanks @jnsdls , made the update so its good to go! Will re-ping @edwardysun to sign off and we can merge it in tomorrow